contrib/export/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	"github.com/tendermint/tendermint/types"
+)
+
+// TODO accept those as parameters in next releases
+var source = "0_34"
+var target = "0_36"
+
+//const map[string]func( extypes.AppMap, ) extypes.AppMap
+
+func main() {
```

Comment:
This should instead be refactored to accept/parse CLI arguments: `--src-version`, `--dst-version`, `--genesis`, `--output`.

Reply:
`source` is used to generate the expected filename - instead the filename should be passed as an argument to the command.
dst version too is required - it should be passed as an argument as well
`output` could just be stdout, so that users can redirect it to a file

Reply:
ok in this moment all resolved except the output + I used shorthand for the tags, shuld I switch to extended ones?

Reply:
`--output` flag IMHO is superfluos. Just print to `stdout` - users can always redirect it to file

Reply:
And yes, I'd replace `-g` with `-genesis`. `-v` is ambiguous too - many programs use it to print version info, how about replace `-v` with `-target`?

Reply:
Agreed, `go run ... --genesis=... --target` or `go run ... --genesis=... --src-version`.

Reply:
We could move the genesis write to some other tool when we will introduce the CCI check, sounds ok? I'll remove it from this PR 👍